Src/BootCamp.Chapter1/Hangman.cs

```diff
@@ -23,18 +21,111 @@ public static class Hangman
+
+            if (!anymousWord.Contains('-'))
+            {
+                return true;
+            }
             return false;
         }
 
+        private static string PlayGame(int lives, string wordToGuess, string anymousWord, string alreadyUsedChars)
+        {
+            do
+            {
+                Console.WriteLine($"You have still {lives} live(s)");
+
+                char input = InputUser();
+
+                var sb = anymousWord.ToCharArray();
+                var counter = 0;
+
+                if (alreadyUsedChars.Contains(input.ToString().ToLower()))
+                {
+                    Console.WriteLine("You already used this char and loose a live");
+                    lives -= 1;
+                    Console.WriteLine($"You have still {lives} live(s)");
+                    continue;
+                }
+
+                for (int i = 0; i < sb.Length; i++)
+                {
+                    if (String.Equals(input.ToString(), wordToGuess[i].ToString(), StringComparison.OrdinalIgnoreCase))
+                    {
+                        sb[i] = input;
+                        counter += 1;
+                        alreadyUsedChars += input.ToString().ToLower();
+                    }
+                }
+
+                anymousWord = new String(sb);
+                Console.WriteLine(anymousWord);
+
+                lives = OutputIfCharacterIsInWord(lives, input, counter);
+            } while (lives != 0 && anymousWord.Contains('-'));
+
+            return anymousWord; 
+        }
+
+        private static int OutputIfCharacterIsInWord(int lives, char input, int counter)
+        {
+            if (counter >= 1)
+            {
+                Console.WriteLine($"the character {input} is {counter} times in the word.{Environment.NewLine}");
+            }
+            else
+            {
+                Console.WriteLine($"The character is not in the word. You loose a live.{Environment.NewLine}");
+                lives -= 1;
+                Console.WriteLine($"You have still {lives} live(s)");
+            }
+
+            return lives;
+        }
+
+        private static char InputUser()
+        {
+            Console.Write("Give me a guess of a character: ");
+            var isValid = char.TryParse(Console.ReadLine(), out char input);
+            while (!isValid)
+            {
+                Console.WriteLine("You only have to enter 1 character");
+                Console.Write("Give me a guess of a character: ");
+                isValid = char.TryParse(Console.ReadLine(), out input);
+            }
+
+            return input;
+        }
+
+        private static string ConvertWordSoUserCannotReadit(string wordToGuess)
+        {
+            var output = "";
+            foreach (var character in wordToGuess)
+            {
+                output += GuessCharacter();
+            }
+
+            return output;
+        }
+
```

Comment:
You already know the length of the word, you don't need to iterate over the wordToGuess.